app/models/manageiq/providers/cisco_intersight/manager_mixin.rb

```diff
@@ -98,6 +97,7 @@ def verify_credentials(args)
     end
 
     def verify_provider_connection(api_client)
+      require 'intersight_client'
```

Comment:
Actually this one isn't needed either technically (but I would keep it) because it's called form line 96 above, which in turn, calls raw_connect, which already requires the gem.

Reply:
Agreed.  I don't think any of these are `required` 🥁 but since we're using the client's internal exception, I like keeping it here.